### PR TITLE
WIP: Added MVP of kube-arbitrator.

### LINKED ISSUE #37: Defined MVP of kube-arbitrator
Currently, there're several features in kube-arbitrator's roadmap; and other requirements from community and user. But we can not build them in one or two days, so I open this issue to trace discussion of kube-arbitrator MVP.

The MVP will be the first version of kube-arbitrator; if any requirements to MVP, please feel free to ask :).

### PR DESCRIPTION
**Pull request by k82cn**
fixes https://github.com/kubernetes-incubator/kube-arbitrator/issues/37

### CODE REVIEW

File: doc/design/mvp.md [hunk 1]
```diff
@@ -0,0 +1,85 @@
...
+
+## infra (MVP)
+
+### code generated
+
+1. Generate Deepcopy: **Done**
+
+### test-infra
+
+kube-arbitrator focus on policy based resource sharing, so unit test and integration test is enough.
```
Comment (yastij) [inline doc/design/mvp.md:18]: @k82cn - From what i read, this won't make usage `kubernetes/test-infra` and that's okay. But is there a plan to provide e2e tests ?

Comment (k82cn) [reply doc/design/mvp.md:18]: Will not provide e2e tests in MVP, as we focus on policy, almost no feature interaction with other components. In the future, e2e is required.

Comment (yastij) [reply doc/design/mvp.md:18]: okay, later do you intend to use same infra. as for `kubernetes/kubernetes` ?

Comment (k82cn) [reply doc/design/mvp.md:18]: hm... not sure now, it dependent on how complex it is. If someone help on it, that'll be great. But integration test framework is required.

**Review by jinzhejz — APPROVED** [review]

**Comment by elaron** [comment]
Has the doc/design/MVP.md been removed? Can't find it in doc folder any more.

**Pull request closed without merging**